devDungeon/src/level/utils/LevelUtils.java

```diff
@@ -55,26 +55,25 @@ public static void explosionAt(
       Coordinate center, int range, long delaySpread, Consumer<Tile> actionPerTile) {
     for (int i = 0; i <= range; i++) {
       final int radius = i;
-      EventScheduler.getInstance()
-          .scheduleAction(
-              () -> {
-                for (int dx = -radius; dx <= radius; dx++) {
-                  for (int dy = -radius; dy <= radius; dy++) {
-                    if (dx * dx + dy * dy > radius * radius) continue; // Ensure circular pattern
+      EventScheduler.scheduleAction(
+          () -> {
+            for (int dx = -radius; dx <= radius; dx++) {
+              for (int dy = -radius; dy <= radius; dy++) {
+                if (dx * dx + dy * dy > radius * radius) continue; // Ensure circular pattern
 
-                    Coordinate target = new Coordinate(center.x + dx, center.y + dy);
-                    if (!hasLineOfSight(center, target)) {
-                      continue; // Skip if no direct line of sight
-                    }
+                Coordinate target = new Coordinate(center.x + dx, center.y + dy);
+                if (!hasLineOfSight(center, target)) {
+                  continue; // Skip if no direct line of sight
+                }
 
-                    Tile tile = Game.currentLevel().tileAt(target);
-                    if (tile != null) {
-                      actionPerTile.accept(tile);
-                    }
-                  }
+                Tile tile = Game.currentLevel().tileAt(target);
+                if (tile != null) {
+                  actionPerTile.accept(tile);
                 }
-              },
-              delaySpread * i);
+              }
+            }
```

Comment:
Lesbarkeit: Das ist ganz schön viel Schleifen und so für einen Lambda-Ausdruck - für meinen Geschmack würde sich hier das Auslagern einer privaten Hilfsmethode durchaus lohnen :)

Reply:
ähm eigentlich war die Datei schon so. Es wurde nur alle etwas verschoben weil `.getInstance()` nicht mehr beim `EventScheduler` nötig ist.

Soll ich trotzdem dann das hier in dem PR ändern?

Reply:
nee. mir ist das nur aufgefallen beim review.

mach ein neues ticket dafür auf und diese diskussion zu. dieser pr soll ja nur verschieben :)

Reply:
#1926